### DISCUSSION TITLE
Issues templates escape sequences

### DISCUSSION THREAD

**Discussion by Mesteery** [community/community#11540] [Projects and Issues]
Escape sequences are rendered only if the string is double quoted.

For example, this is not rendered correctly:
```yaml
name: \U0001F41B Bug report
...
```
unlike:
```yaml
name: "\U0001F41B Bug report"
...
```

Is there a particular reason for this? If not, would it be possible to support single quoted strings and unquoted strings?

**Comment by Oleno21** [comment]
> When emojis are written in this way `\U0001F41B`, they are rendered only if the string is double quoted.
> 
> 
> 
> For example, this is not rendered correctly:
> 
> ```yaml
> 
> name: \U0001F41B Bug report
> 
> ...
> 
> ```
> 
> unlike:
> 
> ```yaml
> 
> name: "\U0001F41B Bug report"
> 
> ...
> 
> ```
> 
> 
> 
> Is there a particular reason for this? If not, would it be possible to support single quoted strings and unquoted strings?
